template/cordova/lib/PluginHandler.js

```diff
@@ -105,7 +105,7 @@ var handlers = {
             var type = obj.type;
 
             if(type === 'projectReference') {
-                project.removeProjectReference(path.join(plugin.dir, src), getTargetConditions(obj));
+                project.removeProjectReference(path.join('..', '..', 'plugins', plugin.id, src), getTargetConditions(obj));
```

Comment:
Another thing is that when we add project reference, we use [`targetDir`](https://github.com/apache/cordova-windows/pull/185/files#diff-7bcd02ec7f03ffbb25512bd1f4a0d388L91) attribute when defined. Shouldn't we add it to path here as well?